tests/FSharp.Test.Utilities/ScriptingShims.fsx

```diff
@@ -2,13 +2,7 @@ namespace global
 
 [<AutoOpen>]
 module GlobalShims = 
-
-    let errorStringWriter = new System.IO.StringWriter()        
-    let oldConsoleError = System.Console.Error
-    do System.Console.SetError(errorStringWriter)
-
     let exit (code:int) = 
-        System.Console.SetError(oldConsoleError)
-        if code=0 then 
+        if code = 0 then 
             () 
-        else failwith $"Script called function 'exit' with code={code} and collected in stderr: {errorStringWriter.ToString()}"
```

Comment:
In case of test failure, the full Console.Error and Console.Out is visible in test output? If yes, all good.

Reply:
Yes, capturing of the output is just moved outside of this shim.